README.md

```diff
@@ -546,6 +546,7 @@ Simple deployment of a mail server, e.g. for inexperienced or impatient admins.
 - [Hubzilla](https://hubzilla.org) - Decentralized identity, privacy, publishing, sharing, cloud storage, and communications/social platform. ([Source Code](https://framagit.org/hubzilla/core)) `MIT` `PHP`
 - [HumHub](https://www.humhub.org/) - Flexible kit for private social networks. ([Source Code](https://github.com/humhub/humhub)) `AGPL-3.0` `PHP`
 - [Isso](https://posativ.org/isso/) - Lightweight commenting server written in Python and Javascript. It aims to be a drop-in replacement for Disqus. ([Source Code](https://github.com/posativ/isso)) `MIT` `Python`
+- [kbin](https://kbin.pub/) - Federated content aggregator and microblogging platform. ([Source Code](https://github.com/ernestwisniewski/kbin)) `AGPL-3.0` `PHP`
```

Comment:
> https://github.com/ernestwisniewski/kbin#getting-started

Please change the language tag to `Nodejs/Docker`

Reply:
Thanks for review, but the main language used in the project is PHP actually. Docker is optional.

Reply:
```
$ yarn install
$ yarn build
```

This suggests that Nodejs is needed. 

Docker is still an option so could be added.